flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -687,4 +718,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
     }
 }
 
+extension Dictionary {
+    mutating func mergeDictionary(other: Dictionary) {
+        for (key, value) in other {
+            self.updateValue(value, forKey: key)
+        }
+    }
+}
```

Comment:
Looks good, could we use [`dict.merging`](https://developer.apple.com/documentation/swift/dictionary/merge(_:uniquingkeyswith:)-m2ub) from Swift 4, or the project uses older version?

Reply:
Good point [`a29ac2a` (#1402)](https://github.com/getsentry/sentry-dart/pull/1402/commits/a29ac2aa2583676b4e6daea80fdd1ff1d37451d8)